src/core/network/qgsblockingnetworkrequest.h

```diff
@@ -82,7 +84,7 @@ class CORE_EXPORT QgsBlockingNetworkRequest : public QObject
      *
      * \see post()
      */
-    ErrorCode get( QNetworkRequest &request, bool forceRefresh = false, QgsFeedback *feedback = nullptr );
+    ErrorCode get( QNetworkRequest &request, bool forceRefresh = false, QgsFeedback *feedback = nullptr, bool emptyResponseIsError = true );
```

Comment:
Instead of a bool argument, could we go for a "flags" approach instead? That'd avoid risk of a bloating the signature if we need additional options in future...

Reply:
good idea, done